pkg/vpcmodel/drawioOutput.go

```diff
@@ -136,6 +138,9 @@ func (d *DrawioOutputFormatter) createRouters() {
 		for _, r := range vpcConfig.RoutingResources {
 			if d.showResource(r) {
 				rTn := d.gen.TreeNode(r)
+				if rTn == nil {
+					continue
+				}
```

Comment:
Why was this check not needed before adding the `loadbalancer`?

Reply:
in abstraction mode, we do not show the pip.
we also do not show the fip of the pip
in that case the TreeNode() with the pip will return nil